src/bun.js/test/jest.zig

```diff
@@ -276,6 +296,21 @@ pub const TestRunner = struct {
         };
         this.files.append(this.allocator, .{ .module_scope = scope, .source = logger.Source.initEmptyFile(file_path) }) catch unreachable;
         entry.value_ptr.* = file_id;
+
+        if (this.test_options.test_line_filters.count() > 0) brk: {
+            const abs_path = if (std.fs.path.isAbsolute(file_path)) file_path else blk2: {
+                var path_buf: bun.PathBuffer = undefined;
+                const cwd = bun.getcwd(&path_buf) catch break :brk;
```

Comment:
this is an unnecessary system call in a somewhat hot code path. Use `bun.fs.FileSystem.instance.top_level_dir` instead.